lib/test.gi

```diff
@@ -136,6 +136,11 @@ InstallGlobalFunction(RunTests, function(arg)
     t := Runtime() - t;
     CloseStream(fres);
     CloseStream(s);
+    # Remove explicit references to GAP's root path, so
+    # tests can be run on multiple machines
+    for gaproot in GAPInfo.KernelInfo.GAP_ROOT_PATHS do
+        res := ReplacedString(res, gaproot, "'GAP_ROOT_PATH'");
```

Comment:
How about `GAP_ROOT_PATH/` instead? I'd find that more pleasing to the eye.

Reply:
Note that we already use `GAPROOT` in the kernel for the same purpose, and also in `tst/testspecial`. I would really prefer to keep these consistent. I don't care much which one (GAPROOT or GAP_ROOT_PATH) is used in the end, just as long as the same is used. (I have a slight preference for GAPROOT, but I'll be happy to accept whichever you prefer)

Reply:
So, my thought process was that we shouldn't use GAPROOT, because, as you say, GAP sometimes prints GAPROOT and we might want to be able to tell if it printed GAPROOT or the actual path.

If we decide that's too subtle an issue to bother about, I'm happy to change to GAPROOT.